### PR TITLE
[VEN-2227]: VIP-215 BUSD debt mitigation

### PR DESCRIPTION
**Pull request by kkirka**
If passed this VIP will perform the following actions:

* Authorize the MoveDebtDelegate contract, to borrow USDT or USDC on behalf of the account [0x489A8756C18C0b8B24EC2a2b9FF3D4d447F79BEc](https://bscscan.com/address/0x489A8756C18C0b8B24EC2a2b9FF3D4d447F79BEc)
* Accept the ownership of the MoveDebtDelegate ([Governance](https://bscscan.com/address/0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396) will be the owner of this contract)

### CODE REVIEW

File: simulations/vip-215/simulations.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  let vUSDC: Contract;
+  let vUSDT: Contract;
+
+  let moveDebtDelegate: Contract;
+  let oracle: Contract;
+
+  before(async () => {
+    const provider = ethers.provider;
+    comptroller = new ethers.Contract(COMPTROLLER, COMPTROLLER_ABI, provider);
+    [vBUSD, vUSDC, vUSDT] = await Promise.all(
```
Comment (chechu) [inline simulations/vip-215/simulations.ts:41]: I don't think the constructor `ethers.Contract` is returning a "thenable" object. So, I think we could remove the `Promise.all` statement

File: simulations/vip-215/simulations.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      [VBUSD, VUSDC, VUSDT].map((address: string) => {
+        return new ethers.Contract(address, VBEP20_DELEGATE_ABI, provider);
+      }),
+    );
+    [busd, , usdt, btc] = await Promise.all(
```
Comment (chechu) [inline simulations/vip-215/simulations.ts:46]:
```suggestion
    [busd, usdc, usdt] = await Promise.all(
```

File: simulations/vip-215/simulations.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    oracle = new ethers.Contract(oracleAddress, PRICE_ORACLE_ABI, provider);
+    moveDebtDelegate = new ethers.Contract(MOVE_DEBT_DELEGATE, MOVE_DEBT_DELEGATE_ABI, provider);
+    await pretendExecutingVip(vip215());
+  });
+
+  describe("Post-VIP contracts status", async () => {
+    it("sets Comptroller implementation back to the diamond version", async () => {
+      expect(await comptroller.comptrollerImplementation()).to.equal(DIAMOND_IMPL);
+    });
+
```
Comment (chechu) [inline simulations/vip-215/simulations.ts:62]: We could invoke the shared check `checkCorePoolComptroller` too

Comment (kkirka) [reply simulations/vip-215/simulations.ts:62]: [684ffc2](https://github.com/VenusProtocol/vips/pull/128/commits/684ffc274abb23ac1edce66fd4e7842272228ea0)

File: simulations/vip-215/simulations.ts (hunk 2 above)
Comment (kkirka) [reply simulations/vip-215/simulations.ts]: [684ffc2](https://github.com/VenusProtocol/vips/pull/128/commits/684ffc274abb23ac1edce66fd4e7842272228ea0)

File: simulations/vip-215/simulations.ts (hunk 1 above)
Comment (kkirka) [reply simulations/vip-215/simulations.ts]: [684ffc2](https://github.com/VenusProtocol/vips/pull/128/commits/684ffc274abb23ac1edce66fd4e7842272228ea0)

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/215

**Pull request merged**